README.md

```diff
@@ -54,7 +54,7 @@ Run `mvn package`
 
 #### 4. Run EHRbase
 
-Replace the * with the current version, e.g. `application/target/application-0.8.0.jar`
+Replace the * with the current version, e.g. `application/target/application-0.9.0.jar`
```

Comment:
Were you correcting the version number for the current state or in preparation for the release? In the latter case we would need to change it to `10`.